css-positioning/fixed.css

```diff
@@ -0,0 +1,13 @@
+html {
+  font-size: 22px;
+}
+
+.page-title {
+  font-size: 3rem;
+  text-align: center;
+
+  /* your code here */
+  position: fixed;
+  left: 20rem;
+  top: 2rem;
```

Comment:
If you make the title `width: 100%`, this won't be necessary, since `text-align: center;` has already been applied

Reply:
Updated: changed to:

- position: fixed; at Line 10
- width: 100% at Line 11